AdminPortal/src/styles/Login.module.css

```diff
@@ -0,0 +1,147 @@
+.outer {
+    display: flex;
+    height: 600px;
+
+    /*align-items: baseline;*/
+
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+
+    gap: 10.275rem;
+}
+
+.LAAKTAlogo {
+    
+    width: 445.31px;
+    height: 445.31px;
+    margin-top: -40px;
+    /*position: absolute;
+    left: 147px;
+    top: 206px;*/
+}
+
+.formContainer {
+    
+    width: 30vw;
+    height: 496.64px;
+
+    /*position: absolute;
+    left: 756.71px;
+    top: 247.62px;*/
+
+    /*margin-top: auto;*/
+
+    background: #FFFFFF;
+    box-shadow: 0px 8.32361px 41.6181px rgba(0, 0, 0, 0.25);
```

Comment:
Where do these numbers come from? Why do they have so many digits after the decimal point? Why did you choose `vw` units? I would simplify these CSS classes (use more sensible values) to make it more portable and easier to maintain.

Reply:
have just made a couple of updates, but in general the vw values were derived by calculating the %width they occupy based on the figma design. Have changed width to 36vw (based on figma). Have also removed all of the crazy decimal values.